### PR TITLE
PFM-3480 Estimate & others documents file name issue

### PR DESCRIPTION
**Pull request by vamshikrishnakole-wtt-egov**
PFM-3480 Estimate & others documents file name issue

### CODE REVIEW

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/Expenditure/src/utils/createBillUtils.js [hunk 1]
```diff
@@ -54,6 +54,8 @@ const fetchDeductions = (deductions, tenantId) => {
 
 export const createBillPayload = (data, contract,  docConfigData,workflowDetails) => {
   
+  debugger;
```
Comment (jagankumar-egov) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/Expenditure/src/utils/createBillUtils.js:57]: Remove this @vamshikrishnakole-wtt-egov

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/Expenditure/src/utils/createBillUtils.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ const createDocObject = (document, docType, otherDocFileName="Others", isActive,
     payload_modal.documentUid = document?.[1]?.['fileStoreId']?.['fileStoreId'];
     payload_modal.key = docType;
     payload_modal.additionalDetails = {
-      fileName : document?.[1]?.['file']?.['name'] ? document?.[1]?.['file']?.['name'] :  documentType?.filter(doc=>doc?.name === docType)?.[0]?.code,
+      fileName : document?.[1]?.['file']?.['name'] ? document?.[1]?.['file']?.['name'] :   document?.[0] ?  document?.[0] : documentType?.filter(doc=>doc?.name === docType)?.[0]?.code,
```
Comment (jagankumar-egov) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/Expenditure/src/utils/createBillUtils.js:15]: Use round braces to find the order of execution

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/Expenditure/src/utils/createBillUtils.js (hunk 1 above)
Comment (vamshikrishnakole-wtt-egov) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/Expenditure/src/utils/createBillUtils.js]: @jagankumar-egov Removed

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/Expenditure/src/utils/createBillUtils.js (hunk 2 above)
Comment (vamshikrishnakole-wtt-egov) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/Expenditure/src/utils/createBillUtils.js]: @jagankumar-egov converted nested ternary into ifelse for better clarification and sonar issue also resolved

**Review by jagankumar-egov — APPROVED** [review]

**Pull request merged**